server.js

```diff
@@ -118,51 +118,50 @@ app.post("/api/webhooks/notifications", async (req, res) => {
 
   var ret = false
 
-  // get the notification request from POST body
+  // get notificationItems from body
   const notificationRequestItems = req.body.notificationItems;
 
-  notificationRequestItems.forEach(({ NotificationRequestItem }) => {
-    console.info("Received webhook notification", NotificationRequestItem);
-    
-    if (validator.validateHMAC(NotificationRequestItem, process.env.REACT_APP_ADYEN_HMAC_KEY)) {
-      if (NotificationRequestItem.success === "true") {
-        // Process the notification based on the eventCode
-        if (NotificationRequestItem.eventCode === "AUTHORISATION"){
-          const payment = paymentStore[NotificationRequestItem.merchantReference];
-          if(payment){
-            payment.status = "Authorised";
-            payment.paymentRef = NotificationRequestItem.pspReference;
-          }
+  // fetch first (and only) NotificationRequestItem
+  const notificationRequestItem = notificationRequestItems[0].NotificationRequestItem
+  console.log(notificationRequestItem)
+  
+  if (validator.validateHMAC(notificationRequestItem, process.env.REACT_APP_ADYEN_HMAC_KEY)) {
```

Comment:
We could apply guard clausing here to make the code more readable:

More-indepth [explanation])https://maximegel.medium.com/what-are-guard-clauses-and-how-to-use-them-350c8f1b6fd2)

Example usage:
```js
if (!validator.validateHMAC(notificationRequestItem, process.env.REACT_APP_ADYEN_HMAC_KEY)) {
   // return 
}
```
This will simplify `ret` as well

Reply:
Done